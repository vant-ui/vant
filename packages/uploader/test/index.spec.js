import Uploader from '../';
import demoTest from '../../../test/demo-test';

demoTest(Uploader);
