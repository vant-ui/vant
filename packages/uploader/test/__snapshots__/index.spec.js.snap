// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders uploader correctly 1`] = `
<section class="van-doc-demo-section demo-uploader" style="background:undefined;">
  <section class="van-doc-demo-block">
    <h2 class="van-doc-demo-block__title">基础用法</h2>
    <div class="demo-uploader-container">
      <div class="van-uploader">
        <i class="van-icon van-icon-photograph">
          <!---->
        </i>
        <input type="file" accept="image/*" class="van-uploader__input">
      </div>
    </div>
  </section>
  <section class="van-doc-demo-block">
    <h2 class="van-doc-demo-block__title">设置 input 属性</h2>
    <div class="demo-uploader-container">
      <div class="van-uploader">
        <i class="van-icon van-icon-photograph">
          <!---->
        </i>
        <input type="file" accept="image/gif, image/jpeg" multiple="multiple" class="van-uploader__input">
      </div>
    </div>
  </section>
</section>
`;
